src/Modules/Events/EventListContainer.js

```diff
@@ -1,40 +1,28 @@
-import React, { useState, useEffect, Fragment } from "react";
-import EventListComponent from "./EventListComponent";
-import { EventHandler } from "../../Utils/EventHandler";
-import { API_URL } from "../../Utils/Urls";
-import SpinnerComponent from "../General/SpinnerComponent";
-import axios from "axios";
-
-const EventListContainer = ({ searchData }) => {
+import React, { useState, useEffect, Fragment } from 'react';
+import EventListComponent from './EventListComponent';
+import { EventHandler } from '../../Utils/EventHandler';
+import { API_URL } from '../../Utils/Urls';
+import SpinnerComponent from '../General/SpinnerComponent';
+import axios from 'axios';
+
+const EventListContainer = ({ zipCode }) => {
   const [agencyResponse, setAgencyResponse] = useState(false);
   const [agencyData, setAgencyData] = useState({});
   const [loading, setLoading] = useState(false);
 
   useEffect(() => {
-    const isSearchData = !!searchData.zip_code;
-    const buildSearchData = data => {
-      if (Object.keys(data)[0]) {
-        handleSubmit(data);
-      }
-    };
-
-    if (isSearchData) {
-      buildSearchData(searchData);
+    if (zipCode) {
+      getEvents(zipCode);
     }
-  }, [searchData]);
+  }, [zipCode]);
 
-  const handleSubmit = async payload => {
-    if (payload) {
+  const getEvents = async zip => {
+    if (zip) {
       setLoading(true);
-      const { zip_code, lat, long } = payload;
       try {
-        const resp =
-          lat !== ""
-            ? await axios.get(API_URL.EVENTS_LIST, {
-                params: { zip_code, lat, long },
```

Comment:
Lat & Long were supposed to feed some math that would customize the sort of the agency.event cards by distance (low to high) within a day. This information was fed in by the 'address auto complete' search aid feature, that we later pulled because of universal terrible reviews (sorry everyone!). We should be good to deprecate/pull it for now. Once we have login, we'll want to customize the search results, but that is future state.